src/ember-app.js

```diff
@@ -29,160 +29,31 @@ class EmberApp {
    * @param {Object} [options.sandboxGlobals] - sandbox variables that can be added or used for overrides in the sandbox.
    */
   constructor(options) {
-    let distPath = path.resolve(options.distPath);
-    let config = this.readPackageJSON(distPath);
-
-    this.appFilePath = config.appFile;
-    this.vendorFilePath = config.vendorFile;
-    this.moduleWhitelist = config.moduleWhitelist;
-    this.hostWhitelist = config.hostWhitelist;
-    this.appConfig = config.appConfig;
-
-    if (process.env.APP_CONFIG) {
-      this.appConfig = JSON.parse(process.env.APP_CONFIG);
-    }
-
-    this.html = fs.readFileSync(config.htmlFile, 'utf8');
-
-    this.sandbox = this.buildSandbox(distPath, options.sandbox, options.sandboxGlobals);
-    this.app = this.retrieveSandboxedApp();
-  }
-
-  /**
-   * @private
-   *
-   * Builds and initializes a new sandbox to run the Ember application in.
-   *
-   * @param {string} distPath path to the built Ember app to load
-   * @param {Sandbox} [sandboxClass=VMSandbox] sandbox class to use
-   * @param {Object} [sandboxGlobals={}] any additional variables to expose in the sandbox or override existing in the sandbox
-   */
-  buildSandbox(distPath, sandboxClass, sandboxGlobals) {
-    let Sandbox = sandboxClass || require('./vm-sandbox');
-    let sandboxRequire = this.buildWhitelistedRequire(this.moduleWhitelist, distPath);
-    let config = this.appConfig;
-    function appConfig() {
-      return { default: config };
-    }
-
-    // add any additional user provided variables or override the default globals in the sandbox
-    let globals = {
-      najax: najax,
-      FastBoot: {
-        require: sandboxRequire,
-        config: appConfig
-      }
-    };
-    for (let key in sandboxGlobals) {
-      if (sandboxGlobals.hasOwnProperty(key)) {
-        globals[key] = sandboxGlobals[key];
-      }
-    }
-
-    return new Sandbox({
-      globals: globals
-    });
+    const distPath = path.resolve(options.distPath);
```

Comment:
I put those in to take advantage of my linter. I'm fine with replacing them with `let`.